examples/force-an-org-custom-flows/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "force-an-org-custom-flows",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "vite": "^7.0.4"
+  },
+  "dependencies": {
+    "@clerk/clerk-js": "https://pkg.pr.new/@clerk/clerk-js@6319"
```

Comment:
I'll update this with the latest clerk-js version once I get this PR merged: https://github.com/clerk/javascript/pull/6319